schemes/contacts-schemas.js

```diff
@@ -0,0 +1,11 @@
+import Joi from "joi";
+
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
you should validate email and phone properly

Reply:
I think now will be better, I used pattern for validate phone number and also add validation for email.
Thank you for your remarks.

Reply:
now it`s better